opensrp-tasking/src/main/java/org/smartregister/tasking/model/TaskDetails.java

```diff
@@ -32,7 +31,7 @@
 /**
  * Created by samuelgithengi on 3/20/19.
  */
-public class TaskDetails extends BaseTaskDetails implements Comparable<TaskDetails>, Serializable {
+public class TaskDetails extends BaseTaskDetails implements Comparable<TaskDetails> {
```

Comment:
I am not sure if this has a negative impact on Reveal. I can see that BaseTaskDetails implements this. Can we assume that this should still work OK?

Reply:
yeah it should